libs/server/Resp/ACLCommands.cs

```diff
@@ -18,53 +18,25 @@ namespace Garnet.server
     internal sealed unsafe partial class RespServerSession : ServerSessionBase
     {
         /// <summary>
-        /// Processes subcommands for the "ACL" RESP-Command.
-        /// NOTE: This function assumes the ACL token was already parsed.
+        /// Processes ACL LIST subcommand.
         /// </summary>
         /// <param name="bufSpan">The remaining command bytes</param>
         /// <param name="count">The number of arguments remaining in bufSpan</param>
         /// <returns>true if parsing succeeded correctly, false if not all tokens could be consumed and further processing is necessary.</returns>
-        private bool ProcessACLCommands(ReadOnlySpan<byte> bufSpan, int count)
```

Comment:
Great work separating these into individual methods. 
I think we are missing ACL GENPASS, LOG and DRYRUN.
Do you think any of those will be useful to add now or later?
https://redis.io/docs/latest/commands/acl-genpass/

Reply:
I think we wait, as tempted as I am.  PR is big enough already.